terraform/gitops/generate-files/templates/pm4ml/values-pm4ml.yaml.tpl

```diff
@@ -54,7 +55,6 @@ mojaloop-payment-token-adapter:
 management-api:
   serviceAccountName: ${pm4ml_service_account_name}
   env:
-    CACHE_URL: redis://${redis_host}:${redis_port}

```

Comment:
Can you put back this param just in case?

Reply:
I have put it back.